extensions/amp-story/amp-story.md

```diff
@@ -1136,6 +1135,42 @@ You can apply multiple entrance animations on one element (for example, an eleme
 If a composed animation is supposed to start after the end of a separate element's animation, make sure that all nested elements that compose the animation have the attribute `animate-in-after` set to the same `id`.
 {% endcall %}
 
+## Optional customization
+
+### `data-text-background-color`
```

Comment:
Could we move this right before the `Links in amp-story-grid-layer` section? I think it'd make sense to have it by the links customization

Reply:
Yeah I couldn't figure out where to put it because it works in `amp-story-cta-layer` and `amp-story-page-attachment` too. I can put it in `amp-story-grid-layer` and mention that in the doc.

Reply:
Done!